benchmark/infrastructure/generated/README.md

```diff
@@ -1,25 +1,13 @@
 *** How to update the benchmark configuration shorthand grammar***
 
-All files in this folder except for `BenchmarkConfigurationShorthandAutomatic.g4` are automatically generated.
-Please leave them untouched. In case the benchmark configuration shorthand grammar has to be changed (e.g. to support a language extension), the workflow is as follows:
+All files in this folder except for `BenchmarkConfigurationShorthandAutomatic.g4` are automatically generated. Please leave them untouched. In case the benchmark configuration shorthand grammar has to be changed (e.g. to support a language extension), the workflow is as follows:
 
-*Change the `BenchmarkConfigurationShorthandAutomatic
-     .g4` file according to your needs
-     .(There is a lot of Documentation on the ANTLR4 format online)
-     .
+- Change the `BenchmarkConfigurationShorthandAutomatic.g4` file according to your needs.(There is a lot of Documentation on the ANTLR4 format online.)
 
-         *Inside this directory,
-    Download the ANTLR parser generator
-``` wget http
-    :  // www.antlr.org/download/antlr-4.11.1-complete.jar
-``` *Inside this directory,
-       Run ANTLR
-``` java - Xmx500M
-           - cp "./antlr-4.11.1-complete.jar:$CLASSPATH" org.antlr.v4.Tool
-           - Dlanguage = Cpp BenchmarkConfigurationShorthandAutomatic.g4
-```
+- Inside this directory, download, or install, the ANTLR parser generator: `https://github.com/antlr/antlr4/blob/master/doc/getting-started.md`
 
-                               * If necessary,
-             adapt the `BenchmarkConfigurationShorthandVisitor.h
-                 /.cpp` (one folder above)*Commit all the changed
-                  files(except the `antlr.jar`)
+- Generate new parser and lexer for `cpp`, following: `https://github.com/antlr/antlr4/blob/master/doc/getting-started.md#generating-parser-code`
```

Comment:
Why isn't the concrete call to ANTLR there anymore? I liked it when it was, especially it documents, which version of ANTLR you used to create the generated files. I would appreciate it if the `wget` and `java` calls were still inside this `README`.